openlibrary/templates/books/add.html

```diff
@@ -68,8 +68,12 @@ <h1>$_("Add a book to Open Library")</h1>
                 <div class="label">$_("Since you are not logged in, please satisfy the reCAPTCHA below.")</div>
                 <div class="g-recaptcha" data-sitekey="$:recaptcha.public_key"></div>
             </div>
-
+        
         <div class="formElement bottom">
+            <br/>
+            <div class="cclicense">
+                $:_('By saving a change to this wiki, you agree that your contribution is given freely to the world under <a href="https://creativecommons.org/publicdomain/zero/1.0/" target="_blank" title="This link to Creative Commons will open in a new window">CC0</a>. Yippee!')
+            </div>
             <div class="input">
                 <br/>
```

Comment:
```suggestion

```
Let's remove the line break before the buttons.  There is too big of a gap between the disclaimer and the button:

![Screenshot from 2021-09-02 11-31-19](https://user-images.githubusercontent.com/28732543/131873367-710b7c07-2fff-4abf-ba4b-bdb53b0b375d.png)